ShestakUI/Modules/Skins/Blizzard/EncounterJournal.lua

```diff
@@ -111,6 +111,13 @@ local function LoadSkin()
 	T.SkinScrollBar(EncounterJournalEncounterFrameInstanceFrameLoreScrollFrameScrollBar)
 	T.SkinScrollBar(EncounterJournalEncounterFrameInfoOverviewScrollFrameScrollBar)
 	T.SkinScrollBar(EncounterJournalEncounterFrameInfoBossesScrollFrameScrollBar)
+	
+	EncounterJournalSuggestFrame.Suggestion1.reward:StripTextures()
+	EncounterJournalSuggestFrame.Suggestion1.reward:SetTemplate("Transparent")
```

Comment:
What about EncounterJournalSuggestFrame.Suggestion2.reward or EncounterJournalSuggestFrame.Suggestion3.reward?
There are icons?
May be you should put code under for i = 1, AJ_MAX_NUM_SUGGESTIONS do